docs/sources/setup-grafana/configure-security/configure-authentication/azuread/index.md

```diff
@@ -65,19 +65,57 @@ To enable the Azure AD/Entra ID OAuth, register your application with Entra ID.
+          - Audience: The audience value that must appear in the external token. For Public cloud, it would be `api://AzureADTokenExchange`. See mentioned documentation for the full list of available audiences.
 
-     1. You must have set `client_authentication` under `[auth.azuread]` to `managed_identity` in the Grafana server configuration for this to work.
+       1. Click **Add**, and then copy the Managed Identity Client ID and the federated credential Audience values. This is your OAuth 2.0 federated credential.
+
+       1. You must have set `client_authentication` under `[auth.azuread]` to `managed_identity` in the Grafana server configuration for this to work.
+
+       1. You may optionally set `workload_identity_token_file` (env var `GF_AUTH_AZUREAD_WORKLOAD_IDENTITY_TOKEN_FILE`) under `[auth.azuread]` to `/var/run/secrets/azure/tokens/azure-identity-token` in the Grafana server configuration. This setting defaults to `/var/run/secrets/azure/tokens/azure-identity-token`.
+
+       {{< admonition type="note" >}}
+       Managed identities as federated credentials are only applicable to workloads hosted in Azure.
+
+       You can only add user-assigned managed identities as federated credentials on Entra ID applications.
+       {{< /admonition >}}
+
+     - **_Workload Identity (K8s/AKS)_**
+
+       1. Refer to [Federated identity credential for an Azure AD application](https://azure.github.io/azure-workload-identity/docs/topics/federated-identity-credential.html#azure-portal-ui) for a complete guide on setting up a federated credential for workload identity.
+          Add a new entry under Federated credentials with the following configuration.
+
+          - Federated credential scenario: Select **Kubernetes accessing Azure resources**.
+          - [Cluster issuer URL](https://learn.microsoft.com/en-us/azure/aks/use-oidc-issuer#get-the-oidc-issuer-url): The OIDC issuer URL that your cluster is integrated with. For example: `https://{region}.oic.prod-aks.azure.com/{tenant_id}/{uuid}`.
+          - Namespace: Namespace of your Grafana deployment. For example: `grafana`.
+          - Service account name: Service account name of your Grafana deployment. For example: `grafana`.
+          - Subject identifier: The expected identity (subject claim) from the OIDC token, which Azure uses to validate and authorize token issuance to the requesting workload. For example: `system:serviceaccount:grafana:grafana`.
+          - Name: A unique descriptive name for the credential.
+          - Description: Grafana OAuth.
+          - Audience: The audience value that must appear in the external token. For Public cloud, it would be `api://AzureADTokenExchange`. See mentioned documentation for the full list of available audiences.
+
+       1. You must have set `workload_identity_enable` (env var `GF_AUTH_AZUREAD_WORKLOAD_IDENTITY_ENABLED`) under `[auth.azuread]` to `true` in the Grafana server configuration for this to work.
```

Comment:
If the `workload_identity_enabled` setting is removed then please remove this line as well.